RDoc-1554 (#1032)
ravendb/docs: Documentation/4.1/Raven.Documentation.Pages/indexes/javaScript-indexes.dotnet.markdown

        })
{CODE-BLOCK/}

{WARNING: `this` Keyword}

> `this` is bound to the state of our JavaScript interpreter, which has no relevance to creating indexes. Using `this` in your index may cause it to break. 

Reviewer: ```suggestion
`this` is bound to the state of our JavaScript interpreter, which has no relevance to creating indexes. Using `this` is unsupported and may cause undefined behavior. 
```
